src/library_glfw.js

```diff
@@ -419,7 +419,7 @@ var LibraryGLFW = {
         GLFW.active.height = GLFW.active.storedHeight;
       }
 
-      Browser.setCanvasSize(GLFW.active.width, GLFW.active.height);
+      Browser.setCanvasSize(GLFW.active.width, GLFW.active.height, true);
```

Comment:
Apparently this line is there because Browser.updateCanvasDimensions always maintains the aspect ratio (```var factor = Math.min(screen.width / w, screen.height / h);```) , which is not the desired behavior (at least in GLFW). By setting the canvas size again in the EventListener this behavior is overriden.

Should I change the behavior of Browser.updateCanvasDimensions so that it does not preserve the aspect ratio instead?